pkg/generate/git/repository.go

```diff
@@ -278,7 +300,7 @@ func (r *repository) GetInfo(location string) (*SourceInfo, []error) {
 	git := func(arg ...string) string {
 		stdout, stderr, err := r.git(location, arg...)
 		if err != nil {
-			errors = append(errors, fmt.Errorf("error invoking '%s': %v. Out: %s, Err: %s",
+			errors = append(errors, fmt.Errorf("error invoking 'git %s': %v. Out: %s, Err: %s",
```

Comment:
`'git'` is added to the start of the args in the `r.git` function, so this error message didn't look right without it since the args it used didn't contain the actual executible